frontend/src/components/layout/navigation/Navigation.tsx

```diff
@@ -38,9 +38,32 @@ export const Navigation = (props: Props) => {
     hasPremium,
     isLoggedIn,
   } = props;
-  const homePath = isLoggedIn ? "/accounts/profile" : "/";
   const isPremiumPage = router.pathname === "/premium";
 
+  const homeLink = isLoggedIn ? (
+    <Link href="/accounts/profile">
+      <a
+        className={`${styles.link} ${styles["home-link"]} 
+          ${styles["hidden-mobile"]} 
+          ${
+            router.pathname === "/accounts/profile" ? styles["is-active"] : ""
+          }`}
+      >
+        {l10n.getString("nav-email-dashboard")}
+      </a>
+    </Link>
+  ) : (
+    <Link href="/">
+      <a
+        className={`${styles.link} ${styles["home-link"]} 
+          ${styles["hidden-mobile"]} 
+          ${router.pathname === "/" ? styles["is-active"] : ""}`}
+      >
+        {l10n.getString("nav-home")}
+      </a>
+    </Link>
+  );
+
```

Comment:
Question (non-blocking): Should this still be called a `homeLink`. Couple of ways to approach this - one being that we render conditionally by checking if user is logged in or we just change the name of the variable. Or we don't do anything and just add a comment. Not important at the moment, but did want to make sure to call it out.

Reply:
Oh! That's a pattern I haven't thought of before, but I like it! Saves a bunch of nesting. Is 5d6fdb96 what you were thinking of?

Tests are a good idea as well, added some in e7eaa30c.